.github/workflows/pr-to-update-choco.yml

```diff
@@ -0,0 +1,98 @@
+name: PR for updating Chocolatey
+
+# This action will run after a tag starting with "v" is published
+on:
+  push:
+    tags:
+      - 'v*'
+  workflow_dispatch:
+
+permissions:
+  contents: read
+
+jobs:
+  choco-update:
+    permissions:
+      contents: write # for Git to git push
+      pull-requests: write # for creating PRs
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Headlamp
+        uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4.1.7
+        with:
+          token: ${{ secrets.KINVOLK_REPOS_TOKEN }}
+          # we need the full history for the git tag command, so fetch all the branches
```

Comment:
by default using the checkout only would grab the latest commit and I believe without it this would break the ability to use git tag which is where we auto fetch the latest tag and set it for use as a env var

![image](https://github.com/user-attachments/assets/6513b470-809a-4d84-9a34-e40149b93b35)

Reply:
ok.